test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetListPackageTests.cs

```diff
@@ -816,6 +820,140 @@ public async Task RunDotnetListPackage_WithMultipleHttpSourcesWithoutAllowInsecu
             result.AllOutput.Should().Contain(expectedError);
         }
 
+        [PlatformFact(Platform.Windows)]
+        public async Task DeprecatedOption_WithDirectPackageReferenceDeprecated_Succeeds()
+        {
+            // Arrange
+            using var pathContext = _fixture.CreateSimpleTestPathContext();
+            var projectA = XPlatTestUtils.CreateProject("ProjectA", pathContext, "net472");
+
+            var packageA100 = new SimpleTestPackageContext("A", "1.0.0");
+            await SimpleTestPackageUtility.CreatePackagesAsync(pathContext.PackageSource, packageA100);
+
+            using var mockServer = new FileSystemBackedV3MockServer(pathContext.PackageSource);
+            var packageSource = new PackageSource(mockServer.ServiceIndexUri, "source");
+            pathContext.Settings.RemoveSource(packageSource.Name);
+            pathContext.Settings.AddSource(packageSource.Name, packageSource.Source, allowInsecureConnectionsValue: "true");
+            mockServer.Start();
+
+            mockServer.DeprecatedPackages.Add(packageA100.Identity);
+
+            _fixture.RunDotnetExpectSuccess(Directory.GetParent(projectA.ProjectPath).FullName, $"add package A --version 1.0.0", testOutputHelper: _testOutputHelper);
+
+            // Act
+            var result = _fixture.RunDotnetExpectSuccess(Directory.GetParent(projectA.ProjectPath).FullName, $"list package --deprecated", testOutputHelper: _testOutputHelper);
+
+            // Assert
+            string[] lines = result.AllOutput.Split([Environment.NewLine], StringSplitOptions.RemoveEmptyEntries);
+            lines.Should().Contain(DirectHeading);
+            var index = Array.IndexOf(lines, DirectHeading) + 1;
+            lines[index].Should().StartWith("   > A                    1.0.0       1.0.0      CriticalBugs");
```

Comment:
nit: Not specific to this PR but I'm wondering if we should be using `/t` or something like that instead of spaces.

Reply:
Potentially.
I'm sure there's some guidance for the .NET SDK output in general.